src/mame/video/ms32.cpp

```diff
@@ -431,24 +417,22 @@ u32 ms32_state::screen_update_ms32(screen_device &screen, bitmap_rgb32 &bitmap,
 	}
 
 	// tile-sprite mixing
-	/* this mixing isn't 100% accurate, it should be using ALL the data in
+	/* this mixing is nowhere near as accurate, it should be using ALL the data in
```

Comment:
Nowhere near as accurate as what?

Reply:
Almost 260 lines of mixing code seems accurate to you? ;p